### DISCUSSION TITLE
Welcome to PyTomography Discussions!

### DISCUSSION THREAD

**Discussion by carluri** [PyTomography/PyTomography#76] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jjgomezcadenas** [comment]
Hi There.  Just started to look into your code and the docs.  The installation instruction propose:

conda install -c conda-forge pytorch

But torchvision is also needed. It could be installed with pip

pip3 install torch torchvision

Thanks
